src/scanner.c

```diff
@@ -42,42 +72,76 @@ bool tree_sitter_rust_external_scanner_scan(void *payload, TSLexer *lexer,
   while (iswspace(lexer->lookahead)) lexer->advance(lexer, true);
 
   if (
-    valid_symbols[RAW_STRING_LITERAL] &&
+    valid_symbols[RAW_STRING_LITERAL_OPEN] &&
     (lexer->lookahead == 'r' || lexer->lookahead == 'b')
   ) {
-    lexer->result_symbol = RAW_STRING_LITERAL;
+    lexer->result_symbol = RAW_STRING_LITERAL_OPEN;
     if (lexer->lookahead == 'b') advance(lexer);
     if (lexer->lookahead != 'r') return false;
     advance(lexer);
 
-    unsigned opening_hash_count = 0;
+    state->opening_hash_count = 0;
+
     while (lexer->lookahead == '#') {
       advance(lexer);
-      opening_hash_count++;
+      state->opening_hash_count++;
     }
 
     if (lexer->lookahead != '"') return false;
+
     advance(lexer);
+    return true;
+  }
+
+  if (
+    valid_symbols[RAW_STRING_LITERAL_CONTENT]
+  ) {
```

Comment:
Slight nitpicks but don't be overzealous with newlines/spacing stuff out, it's a bit cleaner if it's one condition to just have it on the same line, and same for the newlines below with the `}`s

Reply:
Fixed